docs/docs/changelogs/v15.1.0.md

```diff
@@ -0,0 +1,99 @@
+---
+title: v15.1.0
+order: 1
+---
+
+This document outlines API breaking changes for that were introduced in `gaia v15.1.0`.
```

Comment:
Also, add a link to the API breaking section of CHANGELOG.md -- https://github.com/cosmos/gaia/blob/release/v15.1.x/CHANGELOG.md#api-breaking

Reply:
added